DESCRIPTION.md

```diff
@@ -8,9 +8,16 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
 
 # Release Notes
 
-- v2.9.1(unreleased)
+- v2.9.1(January 23, 2023)
 
+  - Fixed a bug where write_pandas did not use the schema and database provided to it
```

Comment:
```suggestion
  - Fixed a bug where write_pandas did not use the schema and database provided to it when creating intermediate objects (Credit @jekaterinakletnaja)
```

Reply:
Sorry, calling out this contributor by handle wouldn't be fair to all of the previous contributors.
```suggestion
  - Fixed a bug where write_pandas did not use user-specified schema and database to create intermediate objects
```